src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+import io.appwrite.services.Teams
+
+val client = Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+
+val teams = Teams(client)
+
+// Create membership with email invite
+val response = teams.createMembership(
+    teamId = "team_id",
+    roles = listOf("developer"),
+    email = "user@example.com"
+)
+```
+{% /multicode %}
+
+# Server-side custom flows {% #server-side %}
+
+Server-side membership creation bypasses the email invitation process, allowing direct member addition. This approach:
+1. Creates an active membership immediately
+2. Doesn't require user acceptance
+3. Gives you complete control over the invitation workflow
+
+Server-side flows give you complete control over the membership process.
+This makes them perfect for scenarios requiring custom workflows, such as bulk user management, automated team assignments, or integration with external systems.
+Since memberships are created directly, users gain immediate access without waiting for email acceptance.
+
+{% multicode %}
+```server-nodejs
+const sdk = require('node-appwrite');
+
+const client = new sdk.Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>')
+    .setKey('<API_KEY>');
+
+const teams = new sdk.Teams(client);
+
+// Create membership directly with userId
+const membership = await teams.createMembership(
+    'team_id',
+    ['developer'],
+    'user_id',
+    'John Doe'  // optional name
+);
+```
+```server-python
+from appwrite.client import Client
+from appwrite.services.teams import Teams
+
+client = Client()
+client.set_endpoint('https://cloud.appwrite.io/v1')
+client.set_project('<PROJECT_ID>')
+client.set_key('<API_KEY>')
+
+teams = Teams(client)
+
+# Create membership directly with userId
+membership = teams.create_membership(
+    team_id='team_id',
+    roles=['developer'],
+    user_id='user_id',
+    name='John Doe'  # optional
+)
+```
+```server-swift
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+    .setKey("<API_KEY>") // Your secret API key
+
+let teams = Teams(client)
+
+// Create membership directly with userId
+let membership = try await teams.createMembership(
+    teamId: "team_id",
+    roles: ["developer"],
+    userId: "user_id",
+    name: "John Doe" // optional
+)
+```
+```server-kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+    .setKey("<API_KEY>") // Your secret API key
+
+val teams = Teams(client)
+
+// Create membership directly with userId
+val response = teams.createMembership(
+    teamId = "team_id",
+    roles = listOf("developer"),
+    userId = "user_id",
+    name = "John Doe" // optional
+)
+```
+{% /multicode %}
+
```

Comment:
It might be useful to have a section on permissions.

Reply:
@stnguyen90 I've added a permissions section (Manage team permissions)